packages/synapse-react-client/src/lib/style/base/_core.scss

```diff
@@ -1321,10 +1320,10 @@ main,
 
 .SRC-tag {
   border: 1px solid SRC.$primary-action-color;
-  border-radius: 100px;
+  border-radius: SRC.$button-border-radius;
   padding: 5px 10px;
   color: SRC.$primary-action-color;
-  text-transform: uppercase;
+  text-transform: SRC.$button-text-transform;
 }
```

Comment:
Style span to match button style, see "Antibody" in FeaturedToolsList: 
main | feature
:----:|:----:
<img width="1214" alt="main_FeaturedToolsList" src="https://user-images.githubusercontent.com/26949006/231309323-4d6b4015-fcae-4b17-ae44-b8737cba2030.png"> | <img width="1195" alt="feature_FeaturedToolsList" src="https://user-images.githubusercontent.com/26949006/231309314-782169d0-c898-4336-8ced-7e23550b1eab.png">

Reply:
These aren't really buttons, so I'm not sure if these should change, please check with Ljubo if you haven't already

Reply:
You're right -- Ljubo confirmed that these should be left as they are for now, but opened an [issue](https://sagebionetworks.jira.com/browse/PORTALS-2625) to figure out a better visual in the future